agents/node/vcxagent-core/test/utils/faber.js

```diff
@@ -128,45 +128,6 @@ module.exports.createFaber = async function createFaber () {
     await vcxAgent.agentShutdownVcx()
   }
 
-  async function createCredDef (revocationDetails, tailsUrl) {
-    revocationDetails = revocationDetails || buildRevocationDetails({ supportRevocation: false })
-
-    await vcxAgent.agentInitVcx()
-
-    logger.info('Faber writing schema on ledger')
-    const schemaId = await vcxAgent.serviceLedgerSchema.createSchema(getSampleSchemaData())
-    await sleep(500)
-
-    logger.info('Faber writing credential definition on ledger')
-    credDefId = getFaberCredDefName()
-    await vcxAgent.serviceLedgerCredDef.createCredentialDefinition(
-      schemaId,
-      credDefId,
-      revocationDetails,
-      tailsUrl
-    )
-    await vcxAgent.agentShutdownVcx()
-  }
-
-  async function buildLedgerPrimitives (revocationDetails, tailsUrl) {
-    await vcxAgent.agentInitVcx()
-
-    logger.info('Faber writing schema on ledger')
-    const schemaId = await vcxAgent.serviceLedgerSchema.createSchema(getSampleSchemaData())
-    await sleep(500)
-
-    logger.info('Faber writing credential definition on ledger')
-    revocationDetails = revocationDetails || buildRevocationDetails({ supportRevocation: false })
-    await vcxAgent.serviceLedgerCredDef.createCredentialDefinition(
-      schemaId,
-      getFaberCredDefName(),
-      revocationDetails,
-      tailsUrl
-    )
-    credDefId = getFaberCredDefName()
-    await vcxAgent.agentShutdownVcx()
-  }
-
   async function buildLedgerPrimitivesV2 (revocationDetails) {
```

Comment:
Perhaps we can drop V2 suffix now (the same for `sendCredentialOfferV2`)? Is there any benefit keeping it around? It might be confusing to see it without corresponding "v1"